src/css/exchange/exchangeDiagnostic.css

```diff
@@ -51,25 +51,29 @@
 }
 
 .overlay-transition {
-  opacity: 0;
-  transition: .5s ease;
-  transform: translate(-50%, -50%);
-  -ms-transform: translate(-50%, -50%)
+    opacity: 0;
+    transition: .5s ease;
+    transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%)
 }
 
 .image {
-  opacity: 1;
-  display: block;
-  width: 100%;
-  height: 100%;
-  transition: .5s ease;
-  backface-visibility: hidden;
+    opacity: 1;
+    display: block;
+    width: 100%;
+    height: 100%;
+    transition: .5s ease;
+    backface-visibility: hidden;
 }
 
 .image-container:hover .image {
-  opacity: 0.3;
+    opacity: 0.3;
 }
 
-.image-container:hover .overlay-transition{
-  opacity: 1;
+.image-container:hover .overlay-transition {
+    opacity: 1;
 }
+
+.ovh-font.ovh-font-0x4 {
```

Comment:
This shouldn't be in this file, you should look for a place in the `ovh-manager-web` project or create a stylesheet for office-attach